fedrec/modules/embeddings.py

```diff
@@ -23,14 +23,20 @@
 def md_solver(n, alpha, d0=None, B=None, round_dim=True, k=None):
     '''
     An external facing function call for mixed-dimension assignment
-    with the alpha power temperature heuristic
-    Inputs:
-    n -- (torch.LongTensor) ; Vector of num of rows for each embedding matrix
+    with the alpha power temperature heuristic.
+
+    Arguments
+    ----------
+    n(torch.LongTensor); Vector of num of rows for each embedding matrix
```

Comment:
Can you please write it like this:
n(torch.LongTensor) -
```suggestion
    n(torch.LongTensor) - Vector of num of rows for each embedding matrix
```
and follow the same across the file.